.github/workflows/pr-lint.yml

```diff
@@ -0,0 +1,42 @@
+name: "Lint PR"
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      # Please look up the latest version from
+      # https://github.com/amannn/action-semantic-pull-request/releases
+      - uses: amannn/action-semantic-pull-request@v3.4.0
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          types: |
+            feat
+            fix
+            chore
+            refactor
+            revert
+          scopes: |
```

Comment:
More of a conceptual thing, but the scopes currently correlate with directories. If we would take topics that are more in line with features we have (e.g. asset / cloud) the commits could give more context hints when debugging.

Reply:
That's probably difficult to scope correctly :D and it is more likely to change.  However, which kind of scopes would you list here then?

Reply:
yes and yes :D I'd use

- cloud
- api
- each language
- tooling
- cli

Now that I listed them they don't seem so different anymore :D

Reply:
Ok, I'll keep it as it is then. We can always come back to this and adjust it if we're not happy with it.